MODULE.bazel

```diff
@@ -71,10 +71,10 @@ darwin_config(
 )
 
 bazel_dep(name = "rules_rust", version = "0.59.2")
-bazel_dep(name = "cxx.rs", version = "1.0.154")
+bazel_dep(name = "cxx.rs", version = "1.0.157")
```

Comment:
We should ignore cxx.rs in `.github/rennovate.json` with `ignoredeps` to make sure it does not get auto-updated `https://docs.renovatebot.com/configuration-options/#ignoredeps

Reply:
Actually we can get rid of the `git_override` because cxx.rs is in the Bazel Central Registry now. https://registry.bazel.build/modules/cxx.rs

Reply:
@louwers fixed (i think), please take a look

Reply:
nope, the registry is several versions behind

Reply:
@nyurik let's use the version that is in the registry?